creator-node/src/utils.js

```diff
@@ -17,6 +17,9 @@ const { generateTimestampAndSignature } = require('./apiSigning')
 const readFile = promisify(fs.readFile)
 
 const THIRTY_MINUTES_IN_SECONDS = 60 * 30
+const TEN_MINUTES_IN_SECONDS = 60 * 10
```

Comment:
since this is going to only be consumed in MS, would make more sense to just define it in MS (don't need to change)